Makefile

```diff
@@ -87,130 +94,113 @@ rm -rf $$TMP_DIR ;\
+.PHONY: vet
+vet: ## Run 'go vet' against code
+	go vet ./...
+
+.PHONY: manager
+manager: generate fmt vet ## Build manager binary
+	go build -o bin/manager main.go
+
+.PHONY: manifests
+manifests: install-controller-gen ## Generate manifests e.g. CRD, RBAC etc.
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=apicurio-registry-operator-role paths="./..." output:crd:artifacts:config=config/crd/resources output:rbac:artifacts:config=config/rbac/resources
 
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+.PHONY: test
 test: generate fmt vet manifests ## Run tests
 	mkdir -p ${ENVTEST_ASSETS_DIR}
 	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
 
-
-manager: generate fmt vet ## Build manager binary
-	go build -o bin/manager main.go
-
-
-run: generate fmt vet manifests ## ??? Run against the configured Kubernetes cluster in ~/.kube/config
-	go run ./main.go
-
-
-install: manifests kustomize ## Install CRDs into a cluster
-	$(KUSTOMIZE) build config/crd | $(CLIENT) apply -f -
-
-
-uninstall: manifests kustomize ## Uninstall CRDs from a cluster
-	$(KUSTOMIZE) build config/crd | $(CLIENT) delete -f -
-
-NAMESPACE ?= "system"
-
-deploy: manifests kustomize ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+.PHONY: deploy
+deploy: manifests install-kustomize ## TODO
 	cd config/manager && $(KUSTOMIZE) edit set image REGISTRY_OPERATOR_IMAGE=${OPERATOR_IMAGE}
-	# $(KUSTOMIZE) build config/build-namespaced | sed "s/\$${NAMESPACE}/${NAMESPACE}/g" | $(CLIENT) apply -f -
-	$(KUSTOMIZE) build config/build-namespaced | $(CLIENT) apply -f -
-
+	cd config/default && $(KUSTOMIZE) edit set namespace ${NAMESPACE}
+	$(KUSTOMIZE) build config/default | $(CLIENT) apply -f - -n ${NAMESPACE}
 
-undeploy: ## Undeploy controller from the configured Kubernetes cluster in ~/.kube/config
-	# $(KUSTOMIZE) build config/build-namespaced | sed "s/\$${NAMESPACE}/${NAMESPACE}/g" | $(CLIENT) delete -f -
+.PHONY: undeploy
+undeploy: ## TODO
 	$(KUSTOMIZE) build config/build-namespaced | $(CLIENT) delete -f -
 
-
-manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=apicurio-registry-operator-role paths="./..." output:crd:artifacts:config=config/crd/resources output:rbac:artifacts:config=config/rbac/resources
-
-
-fmt: ## Run 'go fmt' against code
-	go fmt ./...
-
-
-vet: ## Run 'go vet' against code
-	go vet ./...
-
-
-generate: controller-gen ## ??? Generate code
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
-
-
-docker-build: test ## Build operator docker image
```

Comment:
Suggest uppercase O for Operator in all cases to match Kubernetes standards. Occurs in a few places in this file.